src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,9 +1,270 @@
 import React, { Component } from 'react';
+import ProductPhoto from './ProductPhoto';
+import ProductSize from './ProductSize';
+import ProductBonus from './ProductBonus';
+import ShoppingBag from './ShoppingBag';
 import './ProductDetail.scss';
 
 class ProductDetail extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      productInfoList: [],
+      imageNum: 0,
+      isSize: true,
+      isBonus: true,
+      isShoppingBag: true,
+      size: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('/data/productData.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productInfoList: data.results,
+        });
+      });
+  }
+
+  postProduct = () => {
+    const { productInfoList } = this.state;
+    fetch('http://10.58.7.79:8000/users/signup', {
+      method: 'POST',
+      body: JSON.stringify({
+        product_id: productInfoList.product_id,
+      }),
+    });
+  };
+
+  changeSize = size => {
+    this.setState({ size: size }, this.hideSizePick());
+  };
+
+  hideSizePick = () => {
+    const { isSize } = this.state;
+    this.setState({ isSize: !isSize });
+  };
+
+  hideBonus = () => {
+    const { isBonus } = this.state;
+    this.setState({ isBonus: !isBonus });
+  };
+
+  hideShoppingBag = () => {
+    const { isShoppingBag } = this.state;
+    this.setState({ isShoppingBag: !isShoppingBag });
+  };
+
+  changePhoto = photo => {
+    this.setState({
+      imageNum: photo,
+    });
+  };
+
+  gotoCart = () => {
+    this.props.history.push('/cart');
+  };
+
+  gotoMain = () => {
+    this.props.history.push('/mainPage');
+  };
+
   render() {
-    return <div></div>;
+    const { productInfoList, size, isSize, isBonus, isShoppingBag, imageNum } =
+      this.state;
+    return (
+      <div className="productDetail">
+        <div className="nav">임시</div>
+        <div className="list">
+          <span className="home" onClick={this.gotoMain}>
+            홈
+          </span>
+          &nbsp;&nbsp;/&nbsp;&nbsp;
+          <span className="category">
+            {productInfoList && productInfoList.section}
+          </span>
```

Comment:
```suggestion
          <span className="category">
            {productInfoList.section}
          </span>
```
직접 boolean 값 확인해보시면 아실텐데 배열이나 객체는 비어있더라도 항상 true 값으로 평가됩니다

Reply:
수정하겠습니다~!